README.md

```diff
@@ -107,14 +114,16 @@ client = MontyClient(":memory:")
 ```python
 from montydb import set_storage, MontyClient
 
+
 set_storage("/db/repo", cache_modified=5)  # optional step
 client = MontyClient("/db/repo")  # use current working dir if no path given
+
 # ready to go
 ```
 
 FlatFile config:
 
-```
+```toml
```

Comment:
I think `ini` renders better than `toml`. :)

Reply:
Thanks I'll update

Reply:
Done.

Btw for the last line of the file you can use `<span>  </span>` I think to achieve two nbsp